test/unit/Cardano/Wallet/BinarySpec.hs

```diff
@@ -61,6 +63,17 @@ spec = do
             let decoded = unsafeDeserialiseFromBytes decodeBlock bs
             decoded `shouldBe` block3
 
+        it "should fail to decode a junk block" $ do
+            let junk = mconcat (replicate 100 "junk")
+                decoded = CBOR.deserialiseFromBytes decodeBlock junk
+            decoded `shouldSatisfy` isLeft
+
+        it "should fail to decode a block with block header data" $ do
+            bs <- L8.readFile "test/data/Cardano/Wallet/BinarySpec-block-header-1"
```

Comment:
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

Reply:
Fixed :+1: